python/utilities/MixingTimeResolution_Aux.cpp

```diff
@@ -0,0 +1,14 @@
+
+#include <pybind11/pybind11.h>
+
+#include <goofit/PDFs/physics/MixingTimeResolution_Aux.h>
+
+using namespace GooFit;
+namespace py = pybind11;
+
```

Comment:
MixingTimeResolution has some pure virtual methods in it. So I think you'll need to do what I did in DataSet to make this work. Look at the PyBind docs under virtual functions, too.